src/hooks/useFeedbackDisclosure.ts

```diff
@@ -0,0 +1,66 @@
+import { useDisclosure } from "@chakra-ui/react"
+import { useLocation } from "react-router-dom"
+
+import { LOCAL_STORAGE_KEYS } from "constants/localStorage"
+
+import { useLoginContext } from "contexts/LoginContext"
+
+import { isEditPageUrl, isSpecialPagesUrl } from "utils/pages"
+
+import { useLocalStorage } from "./useLocalStorage"
+
+type LastSeenFeedbackTime = number
+type UserId = string
+
+type FeedbackStorageMappings = Record<UserId, LastSeenFeedbackTime>
+
+const NPS_SURVEY_INTERVAL = 1814400000 as const // 3 weeks - 3 * 7 * 24 * 60 * 60 * 1000
+
+const { REACT_APP_SHOW_NPS_FORM } = process.env
```

Comment:
do remember to parse here, this is not true but "true"

Reply:
@seaerchin I got your point, I still think it's better to control it using string "true" and "false" value, i.e. do a parse here to boolean based on the string value of `process.env.REACT_APP_SHOW_NPS_FORM` because: 

- it's clear when we want to turn on/off the flag, we need to use true/false respectively in env var
- we can identify whether the env var is missing or has unintended string value
- the const here can be guaranteed to be boolean which guarantee `isSurveyRequired` to be boolean on line 52 and saves the cast using `!!` on line 59

Reply:
sure, not a big deal